resources/zib/zib-TobaccoUse.xml

```diff
@@ -0,0 +1,230 @@
+      <alias value="TabakGebruikStatus" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.7.2.2--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-tobaccouse-v3.2-2020EN" />
+        <map value="NL-CM:7.2.10" />
+        <comment value="TobaccoUseStatus" />
+      </mapping>
+    </element>
+    <element id="Observation.note">
+      <path value="Observation.note" />
+      <max value="1" />
+    </element>
+    <element id="Observation.note.text">
+      <path value="Observation.note.text" />
+      <short value="Comment" />
+      <definition value="Relevant comments on the patient’s use of tobacco." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-tobaccouse-v3.2-2020EN" />
+        <map value="NL-CM:7.2.3" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="code" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.component:typeOfTobaccoUsed">
+      <path value="Observation.component" />
+      <sliceName value="typeOfTobaccoUsed" />
+    </element>
+    <element id="Observation.component:typeOfTobaccoUsed.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="53661000146106" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:typeOfTobaccoUsed.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="TypeOfTobaccoUsed" />
+      <definition value="Type of tobacco the patient uses." />
+      <alias value="SoortTabakGebruik" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.7.2.3--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-tobaccouse-v3.2-2020EN" />
+        <map value="NL-CM:7.2.9" />
+        <comment value="TypeOfTobaccoUsed" />
+      </mapping>
+    </element>
+    <element id="Observation.component:amount">
+      <path value="Observation.component" />
+      <sliceName value="amount" />
+    </element>
+    <element id="Observation.component:amount.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="266918002" />
```

Comment:
Where does this code come from? Can't find it on zibs.nl, nor in mapping Excel, nor in a zib ticket. Or am I missing something?

Reply:
https://bits.nictiz.nl/browse/ZIB-562 @jduwel

Reply:
ok, nice! Please make sure that the Zib centre will use this terminology code in the next release. It is currently an answer below a ticket that already is in a release.  
For now, I will resolve this conversation.